packages/react/src/status-badge/StatusBadge.tsx

```diff
@@ -1,48 +1,101 @@
 import { ReactNode } from 'react';
 import { Box } from '../box';
 import { Flex } from '../flex';
-import { boxPalette, mapSpacing, tokens } from '../core';
-import { SuccessIcon, AlertIcon, InfoIcon, WarningIcon } from '../icon';
+import { boxPalette, mapSpacing } from '../core';
+import {
+	AlertCircleIcon,
+	AlertFilledIcon,
+	AlertIcon,
+	HelpIcon,
+	InfoFilledIcon,
+	InfoIcon,
+	ProgressBlockedIcon,
+	ProgressDoingIcon,
+	ProgressPausedIcon,
+	ProgressTodoIcon,
+	SuccessFilledIcon,
+	SuccessIcon,
+	WarningCircleIcon,
+	WarningFilledIcon,
+	WarningIcon,
+} from '../icon';
 import { Text } from '../text';
 
 export type StatusBadgeProps = {
-	/** The visual weight to apply. */
-	weight?: StatusBadgeWeight;
-	/** The status that is printed in the text label. */
+	/** The visual appearance of the badge. */
+	appearance?: StatusBadgeAppearance;
+	/** The text label in the badge. */
 	label: ReactNode;
-	/** The color tone to apply. */
-	tone: StatusBadgeTone;
+	/** @deprecated The color tone to apply. */
+	tone?: StatusBadgeTone;
+	/** Determines the tone and icon of the badge. */
+	type?: StatusBadgeType;
```

Comment:
Changed to a single `type` prop (name TBD) since not all attention levels are applicable to all statuses

Reply:
I don't agree with this approach.

I would still split the values between 2 props, "type/status" (eg. blocked, error, info) and "importance/visibility/emphasis" (high, medium, low)

Throw an error saying "The '____' type does not support the importance prop" if an importance prop is detected.

The importance prop can default to "medium" for the types that support it, otherwise it is equal to "low".

Reply:
No functional difference, but I've changed this to a single `tone` prop now. We want to make the change for consumers as simple as possible, and we want to lean in on tone conveying the meaning of multiple things. It also helps us align Figma, which cannot throw errors if you use an incorrect combination.

Reply:
Ok, fair point about Figma. It isn't the end of the world to have the one `tone` prop control both aspects.